content/docs/tips-and-tricks/azure-guide.md

```diff
@@ -0,0 +1,348 @@
+---
+title: Guide - Run Bank-Vaults stack on Azure
+linktitle: Azure guide
+---
+
+In this guide, you will:
+
+- start an AKS cluster,
+- deploy the [Vault Operator](https://github.com/bank-vaults/vault-operator) and the [Vault Secrets Webhook](https://github.com/bank-vaults/vault-secrets-webhook),
+- create a Vault instance which is configured to use [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/) to store Vault's root and unseal tokens,
+- configure [Azure Storage](https://learn.microsoft.com/en-us/azure/storage/) to persist Vault's data, and
+- configure the [Azure Auth method](https://developer.hashicorp.com/vault/docs/auth/azure) for the Webhook to authenticate against Vault.
+
+## Prerequisites
+
+- Access to Azure cloud with a subscription
+- `azure-cli` installed on your machine
+
+## Step 1: Create Azure resources
+
+First and foremost, ensure that you are logged in to your Azure account with `azure-cli`:
+
+```bash
+az login --tenant <YourTenantName>
+```
+
+Expected output:
+
+```json
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "<YourHomeTenantId>",
+    "id": "<YourSubscriptionId>",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "<YourSubscriptionName>",
+    "state": "Enabled",
+    "tenantId": "<YourTenantId>",
+    "user": {
+      "name": "<YourUserName>",
+      "type": "user"
+    }
+  }
+]
+```
+
+You will need `<YourSubscriptionId>` and `<YourTenantId>` later.
+
+If you don't already have a `Resource group` you would like to use, create a new one:
+
+```bash
+az group create --name "bank-vaults-test-rg" --location "EastUS"
+{...}
+```
+
+### Create an AKS cluster
+
+```bash
+# create cluster
+az aks create --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster" --generate-ssh-keys
+{...}
+
+# write credentials to kubeconfig
+az aks get-credentials --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster"
+
+# if you need to look at cluster information again
+az aks show --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster"
+```
+
+### Create an App Registration and a Client secret
+
+You will use this `App Registration` as the resource for generating MSI access tokens for authentication. A more detailed guide for this can be found [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application-cli-rest).
```

Comment:
```suggestion
This **App Registration** resource will be used for generating MSI access tokens for authentication. A more detailed guide for this can be found [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application-cli-rest).
```